client/header/activity-panel/activity-card/style.scss

```diff
@@ -262,61 +261,6 @@
 	}
 }
 
-.woocommerce-review-activity-card {
```

Comment:
Moved some of this to the homescreen reviews style.scss and deleted the rest as the header review panel is removed.